### PR TITLE
Delete expired deployments from db

### PR DESCRIPTION
**Pull request by samiyaakhtar**
Closes https://github.com/microsoft/bedrock/issues/610

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ This is an initiative to visualize [Project Bedrock](https://github.com/microsof
 
 Currently, Spektate consists of a command line interface and a simple dashboard prototype. The instructions to use both are below.
 
+Note: Spektate dashboard will delete deployments when their corresponding builds/releases have expired in Azure DevOps.
+
```
Comment (edaena) [inline README.md]: nit: should we mention something like "as of january 2020 this is every 2 weeks?" or a link to the expiration policy, if any exists?

Comment (samiyaakhtar) [reply README.md]: @edaena I can't seem to find the page where they specify it, but i'll continue looking for it

**Comment by andrebriggs** [comment]
@samiyaakhtar how did this pass the PR build check but fail on merge to master?

**Comment by samiyaakhtar** [comment]
@andrebriggs I think I was doing a bunch of testing with my other branch and accidentally triggered a build on master, then stopped it midway through. It seems to be passing now

**Pull request merged**